samples/bluetooth/channel_sounding_ras_initiator/prj.conf

```diff
@@ -0,0 +1,39 @@
+#
+# Copyright (c) 2024 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+CONFIG_NCS_SAMPLES_DEFAULTS=y
+CONFIG_DK_LIBRARY=y
+
+CONFIG_BT=y
+CONFIG_BT_SMP=y
+CONFIG_BT_CENTRAL=y
+CONFIG_BT_MAX_CONN=1
+CONFIG_BT_BONDABLE=n
+
+CONFIG_BT_GATT_CLIENT=y
+CONFIG_BT_GATT_DYNAMIC_DB=y
+
+CONFIG_BT_CHANNEL_SOUNDING=y
+CONFIG_BT_RAS=y
+CONFIG_BT_RAS_RREQ=y
+
+CONFIG_BT_SCAN=y
+CONFIG_BT_SCAN_FILTER_ENABLE=y
+CONFIG_BT_SCAN_UUID_CNT=1
+
+# The Ranging Profile recommends a MTU of at least 247 octets.
+CONFIG_BT_L2CAP_TX_MTU=498
+CONFIG_BT_BUF_ACL_TX_SIZE=502
+CONFIG_BT_BUF_ACL_RX_SIZE=502
+CONFIG_BT_ATT_PREPARE_COUNT=3
+
+# This allows CS and ACL to use different PHYs
+CONFIG_BT_TRANSMIT_POWER_CONTROL=y
+CONFIG_BT_CTLR_DATA_LENGTH_MAX=251
+
+CONFIG_HEAP_MEM_POOL_SIZE=1024
+CONFIG_MAIN_STACK_SIZE=4096
```

Comment:
I realize you copied this from my samples, but in hindsight it might not be needed since the big arrays in distance_estimation.c are static now

Reply:
if you still have this open maybe we can do a quick run without this increased stack size